hawkular-wildfly-agent-ansible-playbook/roles/fetch-and-install-hawkular-wildfly-agent/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+---
+- name: Create Hawkular download tmp directory
+  file: path="{{hawkular_download_tmpdir}}" state=directory
+
+- name: Download Hawkular WildFly Agent installer
+
+  # Can't use due to bug: https://github.com/ansible/ansible-modules-core/issues/2166
+  #uri:
+  #  url="{{hawkular_server_url}}/hawkular/wildfly-agent/installer"
+  #  dest="{{hawkular_download_tmpdir}}/hawkular-wildfly-agent-installer.jar"
+  #  user="{{hawkular_server_url_username}}"
+  #  password="{{hawkular_server_url_password}}"
+  #  force_basic_auth=true
+  #  method=POST
+  #  body=""
+  #  follow_redirects=all
+  #  validate_certs=no
+
+  shell:
+    "curl --silent --show-error --create-dirs --user '{{hawkular_server_url_username}}:{{hawkular_server_url_password}}' --output '{{hawkular_download_tmpdir}}/hawkular-wildfly-agent-installer.jar' {{hawkular_server_url}}/hawkular/wildfly-agent/installer"
```

Comment:
We could use ansible to install curl :) but really, this was just a stop-gap for Ansible 1.9. Moving to Ansible 2.0 and I would have been able to use the uri: module instead (I wanted to use uri: module now, but I'm on Ansible 1.9 (Fedora 23 doesn't have 2.0 in its dnf repos) and that bug I mentioned in the comments breaks the uri: module).